app/mailers/claim_mailer.rb

```diff
@@ -2,40 +2,43 @@ class ClaimMailer < ApplicationMailer
   helper :application
 
   def submitted(claim)
-    @claim_description = claim_description(claim)
-    view_mail_with_claim_and_subject(claim, "Your claim #{@claim_description} has been received, reference number: #{claim.reference}")
+    @claim = claim
+    @claim_description = claim_description
+    view_mail_with_claim_and_subject("Your claim #{@claim_description} has been received, reference number: #{claim.reference}")
```

Comment:
It doesn't make much difference, but I think it might be neater to refer to the `@claim` instance variable here (and in other calls to `view_mail_with_claim_and_subject`) now we have the reference.